bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +22,279 @@ class CoffeeLintBear:
+                        no_stand_alone_at: bool=True,
+                        disable_throwing_strings: bool=True,
+                        allow_trailing_semicolons: bool=False,
+                        allow_trailing_whitespaces: bool=False,
+                        no_unnecessary_double_quotes: bool=False,
+                        use_english_operator: bool=False,
+                        spaces_around_operators: bool=False,
+                        space_after_comma: bool=True,
+                        cyclomatic_complexity: int=10,
+                        prevent_duplicate_keys: bool=True,
+                        consistent_line_endings_style: str='',
+                        no_this: bool=False,
+                        no_decr_or_incrementation_operators: bool=False,
+                        no_empty_parameter_list: bool=False,
+                        no_empty_functions: bool=False,
+                        enforce_parentheses_on_non_empty_constructors: bool=True
+                        ):
+        """
+        :param max_line_length:
+            Maximum number of characters per line.
+        :param max_line_length_affect_comments:
+            Determines if ``max_line_length`` should also affects comments or
+            not.
+        :param space_before_and_after_arrow:
+            Determines if spaces should be used before and after the arrow.
+        :param check_braces_spacing:
+            Checks if proper spacing is used inside curly braces.
+        :param braces_spacing_width:
+            Determines the number of blank spaces after the opening ``{`` and
+            before the closing brace ``}`` given that there is something within
+            the braces.
+        :param spacing_in_empty_braces:
+            Determines the number of blank spaces after the opening ``{`` and
+            before the closing brace ``}`` given empty content.
+        :param class_naming_camel_case:
+            Checks whether the classes name should be in camel-case or not.
+        :param spaces_before_and_after_colon:
+            Checks the number of spaces before and after colon.
+        :param spaces_before_colon:
+            Determines the number of blank spaces before colon when
+            ``spaces_before_and_after_colon == True``.
+        :param spaces_after_colon:
+            Determines the number of space after colon when
+            ``spaces_before_and_after_colon == True``.
+        :param enforce_newline_at_EOF:
+            Checks if the file ends with a single newline.
+        :param use_spaces:
+            Forbids tabs in indentation and applies two spaces for this purpose.
+        :param tab_width:
+            Length of the tab for indentation.
+        :param number_of_newlines_after_classes:
+            Determines the number of newlines that separate the class definition
+            and the rest of the code.
+        :param no_backticks:
+            Prevents some JavaScript elements like ``eval`` to affect
```

Comment:
Hm it seems more adverse which is not always the case. For example it won't hurt to embed an object or some other simple stuff. Also it seems like only those nasty elements are prohibited.
Shall we neutralise it and make it more general like `prohibit embedding javascript snippet in coffeescript code` ? Thoughts?